docs/content/docs/reference/verbs.md

```diff
@@ -15,13 +15,24 @@ top = false
 
 ## Defining Verbs
 
+{% code_selector() %}
+
+<!-- go -->
 To declare a Verb, write a normal Go function with the following signature, annotated with the Go [comment directive](https://tip.golang.org/doc/comment#syntax) `//ftl:verb`:
 
+<!-- kotlin -->
+To declare a Verb, write a normal Kotlin function with the following signature, annotated with the Kotlin [annotation](https://kotlinlang.org/docs/annotations.html) `@Verb`:
+
 ```go
```

Comment:
How does this work? souldn't the Go code block be in the `<!-- go -->` section?

Reply:
Code blocks will auto switch as well as comments.

Reply:
I think I'd prefer it if it was just comments, it seems a bit confusing to have them be able to be out of order (like in this example), and if they can't be out of order then there's no need for code blocks to switch.

Reply:
Alright I'll fix it up.